JFrogVSExtension/Components/Component.cs

```diff
@@ -15,23 +15,24 @@ public class Component : BaseViewModel
         public string Artifact { get; set; } = "";
         public string Name { get; set; } = "";
         public string Version { get; set; } = "";
-        public string Type { get; set; } = "Nuget";
+        public string Type { get; set; } = "";
         public List<License> Licenses { get; set; } = new List<License>();
         public Severity TopSeverity { get; set; } = Severity.Normal;
         public List<string> Dependencies { get; set; }
         public List<Issue> Issues { get; set; } = new List<Issue>();
 
         public Component() { }
-        public Component(string artifactId)
+        public Component(Dependency artifact)
```

Comment:
Dependency also contains all the dependencies, so it might be confusing to pass it and not use that value. I would consider only passing the artifactId and packageType to the constructor.